src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -14,7 +14,7 @@
                     {% if profile %}
                         <div id="resume">
                             <a id="resume-view-all" href="{{ path('_profiler_search', {limit: 10}) }}">View last 10</a>
-                            <a id="resume-view-latest" href="{{ path('_profiler', {token: 'latest', 'panel': panel}) }}">View latest</a>
+                            <a id="resume-view-latest" href="{{ path('_profiler', {token: 'latest'} + all_query) }}">View latest</a>
```

Comment:
you should use the ``|merge()`` filter. I don't think using the ``+`` to merge arrays is a supported use case in Twig (it is not a documented one for sure)